02_configure_host.sh

```diff
@@ -37,7 +37,19 @@ fi
 # Allow ipmi to the virtual bmc processes that we just started
 sudo iptables -I INPUT -i virbr0 -p udp -m udp --dport 6230:6235 -j ACCEPT
 
+# Need to route traffic from the provisioning host.
+if [ -z $EXT_IF ]; then
+  sudo iptables -t nat -A POSTROUTING --out-interface ${EXT_IF} -j MASQUERADE
+  sudo iptables -A FORWARD --in-interface virbr0 -j ACCEPT
+fi
+
+# Need to pass the interface for the provisioning network on baremetal
+if [ -z $PRO_IF ]; then
+  brctl addif ${PRO_IF} brovc
+fi
+
 # Switch NetworkManager to internal DNS
 sudo mkdir -p /etc/NetworkManager/conf.d/
 sudo crudini --set /etc/NetworkManager/conf.d/dnsmasq.conf main dns dnsmasq
-sudo systemctl restart NetworkManager
+sudo systemctl reload NetworkManager
```

Comment:
Wasn't this part of a previous PR?

Reply:
Yes it was discussed in https://github.com/metalkube/dev-scripts/pull/61 but @derekhiggins pointed out we probably can't reload in 02, only 05, since at this point when configuring a new host we can't be certain NM is running yet?

Reply:
ack - so can we move the dnsmasq configuration to 05?

Reply:
I think it still needs to be restart here, and now that we're creating a separate upstream.conf we don't need to move this logic to 05 where we do the openshift specific dnsmasq stuff?

Reply:
so, if I revert to restart, this is what happens
```+ sudo mkdir -p /etc/NetworkManager/conf.d/
+ sudo crudini --set /etc/NetworkManager/conf.d/dnsmasq.conf main dns dnsmasq
+ [[ -n 1.1.1.1 ]]
+ echo server=1.1.1.1
+ tee /etc/NetworkManager/dnsmasq.d/upstream.conf
server=1.1.1.1
+ sudo systemctl restart NetworkManager
[root@f13-h02-000-1029u dev-scripts]# ping google.com
ping: google.com: Name or service not known
[root@f13-h02-000-1029u dev-scripts]#
``` 

This is what happens w/ a reload

```
+ sudo mkdir -p /etc/NetworkManager/conf.d/
+ sudo crudini --set /etc/NetworkManager/conf.d/dnsmasq.conf main dns dnsmasq
+ [[ -n 1.1.1.1 ]]
+ echo server=1.1.1.1
+ tee /etc/NetworkManager/dnsmasq.d/upstream.conf
server=1.1.1.1
+ sudo systemctl reload NetworkManager
[root@f13-h02-000-1029u dev-scripts]# ping google.com
PING google.com (172.217.15.110) 56(84) bytes of data.
64 bytes from iad30s21-in-f14.1e100.net (172.217.15.110): icmp_seq=1 ttl=47 time=17.9 ms
64 bytes from iad30s21-in-f14.1e100.net (172.217.15.110): icmp_seq=2 ttl=47 time=17.9 ms

--- google.com ping statistics ---
2 packets transmitted, 2 received, 0% packet loss, time 1001ms
rtt min/avg/max/mdev = 17.913/17.940/17.967/0.027 ms
[root@f13-h02-000-1029u dev-scripts]#```